File: src/main/java/com/estonianport/agendaza/model/Capacidad.java
package com.estonianport.agendaza.model;


import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Entity
@Getter
@Setter
public class Capacidad {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;

	@Column(name = "capacidad_adultos")
	private int capacidadAdultos;

	@Column(name = "capacidad_ninos")
	private int capacidadNinos;

}

File: src/main/kotlin/com/estonianport/agendaza/model/Capacidad.kt
package com.estonianport.agendaza.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id

@Entity
class Capacidad(

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long,

    @Column(name = "capacidad_adultos")
    var capacidadAdultos : Int,

    @Column(name = "capacidad_ninos")
    var capacidadNinos : Int){}